Contribute/style.md

```diff
@@ -5,33 +5,41 @@ title: C Coding Conventions and Style Guide
 pre: Coding Conventions
 pre_link: conventions.html
 sub: '<p>
-  Formatting, style, and C language subset requirements, divided into
-  <a href="#general-c-code">general</a>, <a href="#user-level-code">user-level</a>,
-  <a href="#kernel-code">kernel code</a>, and
-  <a href="#verification-requirements">verification requirements</a>.
+
+  Formatting, style, and C language subset requirements, divided into <a
+  href="#general-c-code">general</a>, <a href="#user-level-code">user-level
+  code</a>, and <a href="#kernel-code">kernel code</a> (including specific
+  requirements for <a href="#verification-requirements">verified kernel
+  code</a>).
```

Comment:
It's for any verified code, not just kernel code

Reply:
Fixed that one up and added more navigation to the page in general. Also tweaked layout and parentheses a bit more throughout.